### PR TITLE
add graphiql index to graphql endpoint

### LINKED ISSUE #143: add graphql web console (graphiql)
currently hard to know how to form queries

### PR DESCRIPTION
**Pull request by willscott**
fix #143

### CODE REVIEW

File: controller/graphql/server.go [hunk 1]
```diff
@@ -10,6 +11,9 @@ import (
 	"github.com/graphql-go/graphql"
 )
 
+//go:embed index.html
+var index embed.FS
```
Comment (mvdan) [inline controller/graphql/server.go:15]: I guess it's easier to use a FS for this, even if it's just one file, so we can use http.FileServer?

**Review by mvdan — APPROVED** [review]
I was about to say that graphiql was a typo for graphql :P

File: controller/graphql/server.go (hunk 1 above)
Comment (willscott) [reply controller/graphql/server.go:15]: right. otherwise it's a custom handler wrapping sending the bytes, right?

**Review by hannahhoward — APPROVED** [review]
LGTM

**Pull request merged**